### PR TITLE
Remove batchnorm script

### PR DESCRIPTION
**Pull request by subutai**

### CODE REVIEW

**Comment by subutai** [comment]
To use it:
        `model = gsc_sparse_cnn(pretrained=True)`
        `model2 = remove_batchnorm(model)`

**Comment by subutai** [comment]
@lscheinkman Could definitely use some help making the core function `remove_batchnorm` more readable.
